scripts/call_contracts.py

```diff
@@ -314,19 +314,16 @@ def whitelist(
     "protocol-fee-ppm",
     type=int,
     required=True,
-    help="Protocol fee percentage described in parts per million",
```

Comment:
Why remove these?

Reply:
> Why remove these?

It's literally in the commit message:
```
scripts: call_contracts: click.argument does not support help text
```
:smile: 

Without the fix, you get an exception at runtime when trying to use that command.